app/ante/vesting.go

```diff
@@ -80,11 +90,34 @@ func (vtd EthVestingTransactionDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx,
 			)
 		}
 
-		// Error if account has locked coins (before surpassing all lockup periods)
-		islocked := clawbackAccount.HasLockedCoins(ctx.BlockTime())
-		if islocked {
-			return ctx, errorsmod.Wrapf(vestingtypes.ErrVestingLockup,
-				"cannot perform Ethereum tx with clawback vesting account, that has locked coins: %s", vested,
+		// Check to make sure that the account does not exceed its spendable balances.
+		// This transaction would fail in processing, so we should prevent it from
+		// moving past the AnteHandler.
+		msgValue := sdk.NewCoin(evmDenom, math.NewIntFromBigInt(msgEthTx.AsTransaction().Value()))
```

Comment:
nit, avoid using `NewCoin` when the denom is already available from the parameters to skip denom validation